### DISCUSSION TITLE
调用vnstation自带的python安装完 vnpy-crypto和 vnpy-binance包后

### DISCUSSION THREAD

**Discussion by codinglong** [veighna-global/vnpy_evo#6] [Q&A]
用vnstation自带的python安装完 vnpy-crypto和 vnpy-binance后,
如何在启动vn-station时加载到这个gateway

**Comment by codinglong** [comment]
需要修改vn-station的启动配置吗, 还是要重新编译一个pythonw.exe...

**Comment by xldistance** [comment]
https://github.com/vnpy/vnpy/blob/dev/examples/veighna_trader/run.py照着这个文件加gateway，用run.py启动
